src/types/pod.ts

```diff
@@ -0,0 +1,14 @@
+export enum PodStatus {
+  // yellow: pod-starting
+  Initializing = "initializing",
```

Comment:
`Initializing` is approximately equivalent to the host's `Uninitialized` state, so I would color it light grey for consistency.